webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -52,10 +57,16 @@ class ManageManifestModal extends Component {
   }
 
   hideModal = () => {
-    this.setState({ showModal: false, showDeleteManifestModalDialog: false });
-    this.props.onClose();
+    this.hideDeleteManifestModal();
+    this.props.setModalClosed({ id: MANAGE_MANIFEST_MODAL_ID });
```

Comment:
`hideDeleteManifestModal` is a function that calls `setModalClosed`. So in `hideModal` `setModalClosed` is called twice, why?

Reply:
@MariaAga There are two modals on this page that I converted to ForemanModal-- the `ManageManifestModal` itself and also the `DeleteManifestModalDialog` which is the confirmation when you delete a manifest.  So, `setModalClosed` is called once for each modal.  Notice the id passed to `setModalClosed` is different in each call.

It's equivalent to 
```javascript
this.props.setModalClosed({ id: DELETE_MANIFEST_MODAL_ID });
this.props.setModalClosed({ id: MANAGE_MANIFEST_MODAL_ID });
```

Reply:
I didn't notice thanks for clearing this up!